deep-learning/_posts/2017-09-07-objDetectorSurvey.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: review
+title: "Speed/accuracy trade-offs for modern convolutional object detectors"
+tags: deep-learning CNN localization essentials survey benchmarking
+author: "Pierre-Marc Jodoin"
+cite:
+    authors: "J Huang, V Rathod, C Sun, M Zhu, A Korattikara, A Fathi, I Fischer, Z Wojna, Y Song, S Guadarrama, K Murphy"
+    title:   "Speed/accuracy trade-offs for modern convolutional object detectors"
+    venue:   "CVPR 2017"
+pdf: "http://openaccess.thecvf.com/content_cvpr_2017/papers/Huang_SpeedAccuracy_Trade-Offs_for_CVPR_2017_paper.pdf"
+---
+
+## Summary
+This is a survey/benchmarking paper focused on localization methods.  This paper is a guide for selecting
+a detection architecture that achieves the right speed/memory/accuracy balance by investigating various ways to
+trade accuracy for speed and memory usage in object detection CNN methods.  In that purpose, they implemented three "meta-architectures":
+
+- SSN (Single Shot Detector)
+- Faster R-CNN
+- R-FCN
+
+different feature extractors :
+
+- Vgg16
+- Resnet-101
+- Inception v2
+- Inception Resnet v2
+- MobileNet
+
+tested the impact of using various number of box proposals (between 10 and 300), and the input image size (300 or 600).
+
+
+## Experimental results
+
+![](/deep-learning/images/objDetector17/sc01.png)
+<div style="text-align:center">
+<img src="/deep-learning/images/objDetector17/sc03.png" width="500">
+</div>
+![](/deep-learning/images/objDetector17/sc04.png)
+
+
+## Conclusion
+
+- Fig.4a shows that **Faster-RCNN + inception ResNet v2** is the top performing configuration.
+- Fig.4b shows that an input resolution of 300 is best for large objects and 600 is best for small objects.
```

Comment:
I disagree. In the plot you can see that green points are in the same region as the blue points on the y axis. Furthermore, the points that are the farthest on the y axis are green. I would conclude that 600 is always better, which seems intuitive. The main drawback must be computation speed.

Reply:
Ok I re-read your sentence and you're probably saying that 300 should only be used when we care about large objects. But at first I understood that 300 is the best resolution for detecting large objects (which is false).